radicle-node/src/main.rs

```diff
@@ -14,6 +14,7 @@ use radicle_node::{logger, service};
 struct Options {
     connect: Vec<(NodeId, Address)>,
     external_addresses: Vec<Address>,
+    daemon: Option<net::SocketAddr>,
```

Comment:
Where is the daemon socket being exposed?  I believed git communication was happening over radicle's connection via 8776 (might be wrong port number), but this suggests a return of the `git url` parameter to the node.